x-pack/plugins/stack_connectors/public/connector_types/servicenow_itsm/servicenow_itsm_params.tsx

```diff
@@ -45,16 +45,31 @@ const defaultFields: Fields = {
 };
 
 const CorrelationIdField: React.FunctionComponent<
-  Pick<ActionParamsProps<ServiceNowITSMActionParams>, 'index' | 'messageVariables'> & {
+  Pick<ActionParamsProps<ServiceNowITSMActionParams>, 'index' | 'messageVariables' | 'errors'> & {
     correlationId: string | null;
+    selectedActionGroupId?: string;
     editSubActionProperty: (key: string, value: any) => void;
   }
-> = ({ index, messageVariables, correlationId, editSubActionProperty }) => {
+> = ({
+  index,
+  messageVariables,
+  correlationId,
+  editSubActionProperty,
+  selectedActionGroupId,
+  errors,
+}) => {
   const { docLinks } = useKibana().services;
   return (
     <EuiFormRow
       fullWidth
       label={i18n.CORRELATION_ID}
+      error={errors['subActionParams.incident.correlation_id']}
+      isInvalid={
+        errors['subActionParams.incident.correlation_id'] !== undefined &&
```

Comment:
I believe keys in this error types definition should've been enums, but this is not this PR's concern.